rfi_file_monitor/applicationwindow.py

```diff
@@ -164,16 +164,45 @@ def __init__(self, *args, **kwargs):
         else:
             self._controls_operations_button.set_sensitive(False)
             self._controls_operations_combo.set_sensitive(False)
-        
+
+        controls_grid_basic.attach(
+            Gtk.Label(
+                label="<b>Remove operation: </b>",
+                use_markup=True,
+                halign=Gtk.Align.END, valign=Gtk.Align.CENTER,
+                hexpand=False, vexpand=False),
+            0, 3, 2, 1)
+
+        self.controls_operations_live = Gtk.ListStore(str, object)
+
+        self.controls_operations_live_combo = Gtk.ComboBox(
+            model=self.controls_operations_live,
+            halign=Gtk.Align.FILL, valign=Gtk.Align.CENTER,
+            hexpand=False, vexpand=False,
+        )
+
+        self.controls_operations_live_combo.pack_start(renderer, True)
```

Comment:
which `renderer` are you using here? I think you're using one created for a different combobox or the tree view??

Reply:
yes I am, I can create a new one